cmd/shcd.go

```diff
@@ -242,45 +245,77 @@ func createSwitchSeed(topology []shcd.ID) error {
 }
 
 // createHMNSeed creates hmn_connections.json using information from the shcd
-func createHMNSeed(topology []shcd.ID) error {
+func createHMNSeed(s *shcd.Shcd) error {
```

Comment:
Changing this to allow new methods to be used on the shcd and keeps the logic in this function (relatively) easier to follow.